plugins/typo_squatter/help.md

```diff
@@ -13,6 +13,10 @@ from threats.
 
 _This plugin does not contain any requirements._
 
+# Supported Product Versions
+
+_There are no supported product versions listed._
```

Comment:
Can you add a support version here?

Reply:
There is no product, so there can't really be a supported product version...

Reply:
Circling back here, it looks like certstream may be a 3rd party product we want to list here or in help.md. Didn't notice it in the trigger until now... sorry! @mrinehart-r7

Reply:
No problem, I will add a supported version for certstream.

Reply:
No problem, I'll add a supported version for certstream.

Reply:
Support version is added. Closing the conversation